### PR TITLE
Dev logout

### PR DESCRIPTION
**Pull request by AndrePaulGrenon**
Logout button functionnality

### CODE REVIEW

File: app/frontend/src/components/TopBar/TopBar.tsx
Comment (oddtiming) [inline app/frontend/src/components/TopBar/TopBar.tsx:1]: So for now logging out only removes the token from the token storage service ?

Comment (JohnKingKong) [reply app/frontend/src/components/TopBar/TopBar.tsx:1]: seems to log you out and brings you back to the auth page aswell

Comment (AndrePaulGrenon) [reply app/frontend/src/components/TopBar/TopBar.tsx:1]: No It clears your self informations and bring you back to loggin page also.

Comment (JohnKingKong) [reply app/frontend/src/components/TopBar/TopBar.tsx:1]: he means the userStatus tho

Comment (oddtiming) [reply app/frontend/src/components/TopBar/TopBar.tsx:1]: Merging it for now, adding a task on the Trello board

**Review by oddtiming — APPROVED** [review]

**Pull request merged**